extensions/notebook/src/book/bookModel.ts

```diff
@@ -215,7 +217,20 @@ export class BookModel {
 		return this._bookItems;
 	}
 
-	public async getSections(tableOfContents: IJupyterBookToc, sections: JupyterBookSection[], root: string, book: BookTreeItemFormat): Promise<BookTreeItem[]> {
+	public set bookItems(bookItems: BookTreeItem[]) {
+		bookItems.forEach(b => {
+			// only add unique notebooks
+			if (this._bookItems.indexOf(b) === -1) {
```

Comment:
Since we aren't using the index here, using this._bookItems.includes() might be more straightforward.